scripts/build/TestImpactAnalysis/tiaf.py

```diff
@@ -18,16 +18,227 @@
 
 logger = get_logger(__file__)
 
+
 class TestImpact:
-    def __init__(self, config_file: str):
+
+    
+
+    def __init__(self, args: dict):
         """
         Initializes the test impact model with the commit, branches as runtime configuration.
 
         @param config_file: The runtime config file to obtain the runtime configuration data from.
+        @param args: The arguments to be parsed and applied to this TestImpact object.
         """
 
+        self._runtime_args = []
+        self._change_list = {"createdFiles": [], "updatedFiles": [], "deletedFiles": []}
         self._has_change_list = False
-        self._parse_config_file(config_file)
+        self._use_test_impact_analysis = False
+
+        # Unique instance id to be used as part of the report name.
+        self._instance_id = uuid.uuid4().hex
+
+        self._s3_bucket = args.get('s3_bucket')
+        self._suite = args.get('suite')
+
+        self._config = self._parse_config_file(args['config'])
+
+        # Initialise branches
+        self._src_branch = args.get("src_branch")
+        self._dst_branch = args.get("dst_branch")
+        logger.info(f"Src branch: '{self._src_branch}'.")
+        logger.info(f"Dst branch: '{self._dst_branch}'.")
+
+        # Determine our source of truth. Also initialises our source of truth property.
+        self._determine_source_of_truth()
+
+        # Initialise commit info
```

Comment:
uber nit (only as I've been burned by this a few times 😅) please use 'American' English for all code (purely for consistency). So this is 'initializes' (more important for code names but good good comments too)